samples/suit/smp_transfer/README.rst

```diff
@@ -240,12 +246,15 @@ After programming the sample to your development kit and updating the sequence n
             .. figure:: /images/suit_smp_select_advanced.png
                :alt: Select ADVANCED
 
-         #. Click on the :guilabel:`READ` button within the **Images** section.
+         #. Click on the :guilabel:`Read` button within the **Images** section.
```

Comment:
```suggestion
         #. Tap on the :guilabel:`Read` button within the *Images* section.
```

Reply:
Could you change any instance of "click" to "tap"? It's a phone app interface

Reply:
OK